contrib/ohi-release/README.md

```diff
@@ -0,0 +1,63 @@
+# 🛠️ `ohi-release`
+
+This is a wrapper of all the steps needed to update the changelog and render a snippet to be ready to be used as a release message. This contribution also includes a script that allow to replicate what this action does locally.
+
+## Example Usage
+
+```yaml
+- uses: newrelic/release-toolkit/contrib/ohi-release@v1
```

Comment:
As the release is supposed to be created in a different step (using the action output) would it make sense to name the action `contrib/ohi-release-notes` instead of `contrib/ohi-release`?

Reply:
Fair enough. Fixed in [#2e447e6](https://github.com/newrelic/release-toolkit/pull/135/commits/2e447e6f41457d446032646f59654b6ce83c963f)